src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,154 @@
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { FindRegistration } from './FindRegistration';
+import { unpublishRegistration } from '../../../api/registrationApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { useDispatch } from 'react-redux';
+import { LoadingButton } from '@mui/lab';
+import { RequiredDescription } from '../../../components/RequiredDescription';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useHistory } from 'react-router-dom';
+import i18n from '../../../translations/i18n';
+import { useMutation } from '@tanstack/react-query';
+import { getRegistrationLandingPagePath } from '../../../utils/urlPaths';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+
+interface DeleteForm {
+  deleteMessage: string;
+}
+
+interface DeletePublicationProps {
+  registration: Registration;
+}
+
+export const DeletePublication = ({ registration }: DeletePublicationProps) => {
+  const [showDeleteButton, setShowDeleteButton] = useState(false);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [selectedDuplicate, setSelectedDuplicate] = useState<Registration | null>(null);
+  const history = useHistory();
+
+  const deleteValidationSchema = Yup.object().shape({
+    deleteMessage: Yup.string()
+      .min(3, t('feedback.validation.must_be_bigger_than', { field: i18n.t('common.justification'), limit: 3 }))
+      .required(t('feedback.validation.is_required', { field: i18n.t('common.justification') })),
+  });
+
+  const unpublishRegistrationMutation = useMutation({
+    mutationFn: (values: DeleteForm) => {
+      if (selectedDuplicate) {
+        return unpublishRegistration(registration.identifier, {
+          type: 'UnpublishPublicationRequest',
+          duplicateOf: selectedDuplicate.id,
+          comment: values.deleteMessage,
+        });
+      } else {
+        return unpublishRegistration(registration.identifier, {
+          type: 'UnpublishPublicationRequest',
+          comment: values.deleteMessage,
+        });
+      }
+    },
+    onSuccess: () => {
+      setShowDeleteModal(false);
+      history.push(`${getRegistrationLandingPagePath(registration.identifier)}?shouldNotRedirect`);
+    },
+    onError: () => {
+      dispatch(setNotification({ message: t('feedback.error.update_registration'), variant: 'error' }));
+    },
+  });
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+        <Divider />
+        {!showDeleteButton && (
+          <IconButton
+            sx={{ alignSelf: 'center' }}
+            title={t('common.show_more_options')}
+            onClick={() => setShowDeleteButton(true)}>
+            <MoreHorizIcon />
+          </IconButton>
+        )}
+        {showDeleteButton && (
+          <Button
+            sx={{ alignSelf: 'center' }}
```

Comment:
Fint om parent styrer layout selv så langt det lar seg gjøre. Virker som alle children skal sentreres her, da er det like greit at parent flex bestemmer det
```suggestion
      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem', alignItems: 'center' }}>
        <Divider flexItem />
        {!showDeleteButton && (
          <IconButton
            title={t('common.show_more_options')}
            onClick={() => setShowDeleteButton(true)}>
            <MoreHorizIcon />
          </IconButton>
        )}
        {showDeleteButton && (
          <Button
```